android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -274,16 +279,19 @@ private void doMenuAction(int menuItemId) {
         FragmentManager fragmentManager = getSupportFragmentManager();
         switch (menuItemId) {
             case R.id.nav_tracks:
+                addShadowToAppBar(true);
                 fragmentManager.beginTransaction()
                         .replace(R.id.content_frame, new TracksFragment(), FRAGMENT_TAG).commit();
                 getSupportActionBar().setTitle(R.string.menu_tracks);
                 break;
             case R.id.nav_schedule:
+                addShadowToAppBar(false);
                 fragmentManager.beginTransaction()
                         .replace(R.id.content_frame, new ScheduleFragment(), FRAGMENT_TAG).commit();
                 getSupportActionBar().setTitle(R.string.menu_tracks);
```

Comment:
![screenshot-area-2016-07-08-032053](https://cloud.githubusercontent.com/assets/6565270/16671164/8a5439be-44bb-11e6-8d8b-b8caa54c03f5.png)
The actionbar title in schedule fragment displays title for tracks.
Please create a new item in strings.xml and use it here.
Rest looks good.
Cheers.